_historical_review/core-tasks/tests/test_reranking_service.py

```
@@ -0,0 +1,473 @@
+        SearchResult(
+            chunk_id="relevant_2",
+            content="Insurance claims processing for California policies",
+            score=0.70,
+            rank=0,
+            method="dense",
+            metadata={}
+        )
+    ]
+
+    # Mock Ollama to return higher scores for relevant docs
+    def mock_generate(model, prompt, stream=False):
```

Ruff (0.14.13)

Unused function argument: `model`

(ARG001)